src/main/java/nl/tudelft/jpacman/game/Game.java

```diff
@@ -104,13 +107,56 @@ public void move(Player player, Direction direction) {
         }
     }
 
+    /**
+     * Store player's status
+     */
+    public static enum PLAYER_STATUS{
+        WIN,
+        LOST,
+        PLAYING
+    }
+
+    /**
+     * Default player's status to playing.
+     */
+    private PLAYER_STATUS playerStatus = PLAYER_STATUS.PLAYING;
+
+    /**
+     * @return The current player's status.
+     * @return
+     */
+    public PLAYER_STATUS getPlayerStatus(){
+        return playerStatus;
+    }
+
+    /**
+     * Set player's status.
+     * @param ps
+     *          Status the player is currently in.
+     */
+    public void setPlayerStatus(PLAYER_STATUS ps){
+        playerStatus = ps;
+    }
```

Comment:
ไม่ควรให้ภายนอกมาเปลี่ยนค่า

```suggestion
    /**
     * Set player's status.
     * @param ps
     *          Status the player is currently in.
     */
    private void setPlayerStatus(PLAYER_STATUS ps){
        playerStatus = ps;
    }
```